integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomainModelInPV.java

```diff
@@ -366,15 +443,28 @@ private static void buildMIIandPushToRepo() {
     checkDirectory(WIT_BUILD_DIR);
     Map<String, String> env = new HashMap<>();
     env.put("WLSIMG_BLDDIR", WIT_BUILD_DIR);
-    createImage(defaultWitParams()
-        .modelImageName(MII_BASIC_IMAGE_NAME)
-        .modelImageTag(miiImageTag)
+    WitParams defaultWitParams = defaultWitParams();
+    if (customWDTHome != null) {
+      defaultWitParams.wdtModelHome(customWDTHome);
+    }
+    createImage(defaultWitParams
+        .modelImageName(imageName)
+        .modelImageTag(imageTag)
         .modelFiles(modelList)
         .wdtModelOnly(true)
         .wdtVersion(WDT_VERSION)
         .env(env)
         .redirect(true));
+    Command.defaultCommandParams()
```

Comment:
looks like this is only for debugging purposes, it can be removed as there is an assert in the next line

Reply:
removed